sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/MetadataConfigurationAsyncClient.java

```diff
@@ -9,7 +9,7 @@
 import com.azure.core.annotation.ServiceMethod;
 import com.azure.core.http.rest.Response;
 import com.azure.security.attestation.implementation.MetadataConfigurationsImpl;
-import com.azure.security.attestation.models.CloudErrorException;
+import com.azure.security.attestation.implementation.models.CloudErrorException;
```

Comment:
You will need to create a public type for this, as it is an exchange type.

Reply:
Yup, missed that one (twice). 
Having said that, I'm not sure that CloudErrorException ever gets raised - it's not a part of any of the other models, which typically raise HttpResultException on the receipt of a non-400 error from the service.

But I'll deal with that when it comes time to actually deal with errors coming from the service. This PR is literally about splitting the autorest code from the implemetnation.